code/modules/antagonists/gang/gang.dm

```diff
@@ -80,6 +80,8 @@
 		have that datum onhand. it would be easier if all of the code the gang team calls on its my_gang_datum was
 		just in the team datum itself, and there were different types of teams instead of different types of gangster
 		that imprint on generic teams, but i'm too lazy to refactor THAT too */
+	var/datum/atom_hud/H = GLOB.huds[ANTAG_HUD_GANGSTER]
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names